### ISSUE TITLE
oocytes from affected individuals could not form pronuclei after fertilization 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9627]
**Preferred term label:**
Lack of oocyte pronucleus formation

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormality of oocyte maturation characterized by the inability of oocytes to form pronuclei after  in vitro fertilization procedures. Oocytes normally show two pronuclei (2PN), 17–20 h after in vitro insemination.
PMID:34552114
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
617996 
PMID: 29606300  4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
